### ISSUE TITLE
deploy private key

### ISSUE THREAD

**Issue by VincentSaelzler** [VincentSaelzler/hyper-homelab#4]
https://blog.confirm.ch/deploying-ssl-private-keys-with-ansible/

**Comment by VincentSaelzler** [comment]
possibly don't do this. using ansible vault is possible but i would need to pass the PW each time the play is run.

for now probably just leaving un-implemented because the only convenience is that i can already auth with github once the vscode envirtonment is ready

**Comment by VincentSaelzler** [comment]
use ansible vault

**Comment by VincentSaelzler** [comment]
may possibly open another issue in case i decide this is a good idea - probably will want to embed secret stuff in images so using ansible vault will be a good skill. however, i don't think i really need the private key deployed.

the only thing i was going to use it for is authenticating with github - still may do but not for now at least
